packages/devtools_app/lib/src/shared/ui/filter.dart

```diff
@@ -552,8 +560,10 @@ class _FilterFieldState<T> extends State<FilterField<T>> with AutoDisposeMixin {
   }
 }
 
-class FilterOptionsDialog<T> extends StatefulWidget {
-  FilterOptionsDialog({
+/// A dialog for editing settings for only the toggleFilters associated with a
+/// [FilterControllerMixin].
+class ToggleFilterOptionsDialog<T> extends StatefulWidget {
```

Comment:
can you add a TODO to try to merge this with FilterDialog since it is duplicating a lot of the same functionality?

Reply:
I honestly didn't notice that, flipping it now :)